templates/logsearch-infrastructure-openstack.yml

```diff
@@ -0,0 +1,63 @@
+compilation:
+  cloud_properties:
+    instance_type: (( grab meta.instance_type.compilation ))
+
+resource_pools:
+- name: elasticsearch_master
+  cloud_properties:
+    availability_zone: (( grab meta.availability_zone ))
+    instance_type: (( grab meta.instance_type.elasticsearch ))
```

Comment:
Would you provide sensible defaults for these fields? Spruce does things differently from spiff, check the aws templates for guidance. Try not to use the "meta convention" where it can be avoided.

https://github.com/geofffranks/spruce

Cheers!